README.md

```diff
@@ -14,7 +14,10 @@ For any changes, please create a feature branch and open a PR for it when you fe
 
 ## Getting Started
 
-This is a Ruby on Rails app that uses Postgres since we'll likely depend on Postgres features around geolocation. Ruby and Rails versions are specified in the project's `.ruby-version` and `Gemfile`.
+This is a Ruby on Rails app that uses Postgres since we'll likely depend on Postgres features around geolocation. 
+This project uses `ruby 2.3.0` and `rails 5.1.1`
+
+Click here to install postress [Postgres installation guide](https://wiki.postgresql.org/wiki/Detailed_installation_guides)
 
```

Comment:
I have submitted a new pull request with a link to the installation instructions and no spelling errors.